plugins/intel/intel-nvme.c

```diff
@@ -58,25 +58,70 @@ struct nvme_additional_smart_log {
 	struct nvme_additional_smart_log_item	host_bytes_written;
 };
 
+#pragma pack(push,1)
```

Comment:
That's a microsoft windows compiler thing. Some new gcc's support it, but let's stick with the native gcc dialect if we can. This structure here is entirely made of u8's, though, so I don't think there's anyway it can be unaligned, right?

Reply:
I was copying the structure used for nvme_additional_smart_log_item that appears to originate from way back at 61fcd8b637cc8a742beafc9e3bd756fd7d8d2dc, but you're right, it doesn't really make sense for this set of u8's.